haproxy_stats.sh

```diff
@@ -1,25 +1,16 @@
 #!/bin/bash
 set -o pipefail
 
-if [[ "$1" = /* ]]
-then
-  HAPROXY_SOCKET="$1"
-  shift 1
-fi
+[ "$#" -gt 3 ] && HAPROXY_SOCK="$(echo $1 | tr -d '\040\011\012\015')" && shift 1
+
+# Define variables and perform checks
+CONFIG_FILE="$(dirname $(readlink -f "$0"))/haproxy_config"
+. $CONFIG_FILE
```

Comment:
I did notice that I ended up using different variable names, but if they are to converge, might as well use more explicit `$HAPROXY_SOCKET`